### DISCUSSION TITLE
app not able to interpret dash bootstrap components

### DISCUSSION THREAD

**Discussion by Hootie9lives** [Coding-with-Adam/Dash-by-Plotly#2] [Q&A]
@Coding-with-Adam,

I am experiencing a problem where the application (in a Linux-based environment, browser is FireFox ) is not able able understand some methods of dash bootstrap components.

For example, as shown below the tabs are not working. 

While this is happening, the same code works when deployed and on my local Windows machine.

We have a need to develop in this Linux environment

![image](https://user-images.githubusercontent.com/28104489/101807437-1c32a380-3ae3-11eb-80da-95d790811ee0.png)

**Comment by Coding-with-Adam** [comment]
Great question, @Hootie9lives 
This is the first time I'm seeing this error, and I don't have Linux. So hopefully someone joining this discussion room can help you. 

In the meantime, you can try to create a post on [plotly forum](https://community.plotly.com/) with this problem or direct message @tcbegley on plotly forum because I think he is one of the creators of Dash Bootstrap.
